app/uk/gov/hmrc/agentsubscription/model/AgencyDetails.scala

```diff
@@ -18,7 +18,7 @@ package uk.gov.hmrc.agentsubscription.model
 import play.api.libs.functional.syntax._
 import play.api.libs.json.{ Json, Reads, _ }
 
-case class BusinessDetails(tradingName: String, businessAddress: OverseasBusinessAddress)
+case class TradingDetails(tradingName: String, tradingAddress: OverseasBusinessAddress)
```

Comment:
Could consider renaming these case classes to all have a prefix of `Overseas`. In particular, the `BusinessContactDetails` could benefit from this as it's very similar to other UK-only case classes.